supabase/migrations/20241216091100_add_size_to_proofs.sql

```diff
@@ -0,0 +1 @@
+alter table public.proofs add column size bigint;
```

Comment:
We should specify a unit here e.g. size_bytes

Reply:
we don't do this with other units (ie time)

Reply:
> _we don't do this with other units (ie time)_

We probably should?